platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx

```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
 
 1. Retrieve the **username** and **password** you received from Seqera support.
 
-2. Run the following Docker command to authenticate to the registry (using the `username` and `password` values copied in step 1):
+2. Authenticate to the registry:
 
    ```bash
    docker login -u '/\<USERNAME\>/' -p '/\PASSWORD\>/' cr.seqera.io
    ```
 
-3. Pull the Seqera container images with the following commands:
+3. Pull the Seqera container images:
 
    ```bash
-   docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.3.0
+   docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.4.3
```

Comment:
We can theoretically start using MDX variables for this now.

Reply:
Sweet! Do we store all the global variables in a single file, or declare them at the start or end of individual MDX pages?